analyzers/tests/SonarAnalyzer.UnitTest/Helpers/ParameterLoaderTest.cs

```diff
@@ -37,7 +37,7 @@ public class ParameterLoaderTest
         public void SetParameterValues_WhenNoSonarLintIsGiven_DoesNotPopulateParameters(string filePath)
         {
             // Arrange
-            var compilation = CreateCompilationWithOption(filePath, SourceText.From(File.ReadAllText("ResourceTests\\SonarLint.xml")));
+            var compilation = CreateCompilationWithOption(filePath, SourceText.From(File.ReadAllText("ResourceTests\\SonarLintXml\\All_properties_small_template\\SonarLint.xml")));
```

Comment:
It's not clear to me what `All_properties_small_template` means - is it that there are no exclusion/inclusion properties?

In case maybe it's possible to be named a bit better so that it immediately gives the idea of what it is about.

Reply:
This file is now removed 👍